server/plugin/plugin.go

```diff
@@ -57,7 +57,8 @@ const (
 )
 
 var (
-	Manifest model.Manifest = root.Manifest
+	Manifest           model.Manifest = root.Manifest
+	testOAuthServerURL                = ""
```

Comment:
```suggestion
    // testOAuthServerURL is the URL for the oauthServer used for testing purposes
    // It should be set through ldflags when compiling for E2E, and keep it blank otherwise
	testOAuthServerURL                = ""
```